cuqi/implicitprior/_regularizedUnboundedUniform.py

```diff
@@ -0,0 +1,66 @@
+from cuqi.implicitprior import RegularizedGaussian
+from cuqi.distribution import Distribution, Gaussian
+
+import numpy as np
+
+
+class RegularizedUnboundedUniform(RegularizedGaussian):

```

Comment:
I was wondering if you had a quick demo you could throw together to illustrate the use of this  implicit prior?

Reply:
I have added a quick demo notebook that shows it, and some of the others, in action.

Reply:
Thanks @jeverink. LGTM!